<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>igwVCRToolBar</class>
 <widget class="QToolBar" name="igwVCRToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VCR Controls</string>
  </property>
  <property name="orientation">
   <enum>Qt::Horizontal</enum>
  </property>
  <property name="PV_MUST_BE_MASTER" stdset="0">
   <bool>true</bool>
  </property>
  <action name="actionVCRPlay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrPlay.svg</normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrPlay.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
  </action>
  <action name="actionVCRReverse">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrReverse.svg</normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrReverse.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Reverse</string>
   </property>
  </action>
  <action name="actionVCRPreviousFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrBack.svg</normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrBack.svg</iconset>
   </property>
   <property name="text">
    <string>Pre&amp;vious Frame</string>
   </property>
   <property name="PV_MUST_BE_MASTER" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionVCRFirstFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrFirst.svg</normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrFirst.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;First Frame</string>
   </property>
  </action>
  <action name="actionVCRNextFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrForward.svg</normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrForward.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Next Frame</string>
   </property>
  </action>
  <action name="actionVCRLastFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrLast.svg</normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrLast.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Last Frame</string>
   </property>
  </action>
  <action name="actionVCRLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrLoop.svg</normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwVcrLoop.svg</iconset>
   </property>
   <property name="text">
    <string>L&amp;oop</string>
   </property>
   <property name="iconText">
    <string>Loop</string>
   </property>
   <property name="toolTip">
    <string>Loop</string>
   </property>
   <property name="statusTip">
    <string>Loop</string>
   </property>
  </action>
  <addaction name="actionVCRFirstFrame"/>
  <addaction name="actionVCRPreviousFrame"/>
  <addaction name="actionVCRReverse"/>
  <addaction name="actionVCRPlay"/>
  <addaction name="actionVCRNextFrame"/>
  <addaction name="actionVCRLastFrame"/>
  <addaction name="actionVCRLoop"/>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
